packages/notifi-core/lib/operations/ConnectWallet.ts

```diff
@@ -0,0 +1,19 @@
+import { ConnectedWallet, Operation, WalletBlockchain } from '../models';
+
+export type ConnectWalletInput = Readonly<{
+  walletPublicKey: string;
+  timestamp: number;
+  signature: string;
+  walletBlockchain: WalletBlockchain;
+  accountId?: string;
```

Comment:
what happens if the account id is undefined?

Reply:
account ID is only required for certain chains!